python/tempo/tsdf.py

```diff
@@ -750,63 +750,63 @@ def asofJoin(
         left_df = self.df
         right_df = right_tsdf.df
 
-        # test if the broadcast join will be efficient
-        if sql_join_opt:
-            spark = SparkSession.builder.getOrCreate()
-            left_bytes = self.__getBytesFromPlan(left_df, spark)
-            right_bytes = self.__getBytesFromPlan(right_df, spark)
-
-            # choose 30MB as the cutoff for the broadcast
-            bytes_threshold = 30 * 1024 * 1024
-            if (left_bytes < bytes_threshold) or (right_bytes < bytes_threshold):
-                spark.conf.set("spark.databricks.optimizer.rangeJoin.binSize", 60)
-                partition_cols = right_tsdf.partitionCols
-                left_cols = list(set(left_df.columns) - set(self.partitionCols))
-                right_cols = list(set(right_df.columns) - set(right_tsdf.partitionCols))
-
-                left_prefix = (
-                    ""
-                    if ((left_prefix is None) | (left_prefix == ""))
-                    else left_prefix + "_"
-                )
-                right_prefix = (
-                    ""
-                    if ((right_prefix is None) | (right_prefix == ""))
-                    else right_prefix + "_"
-                )
+        spark = SparkSession.builder.getOrCreate()
+        left_bytes = self.__getBytesFromPlan(left_df, spark)
+        right_bytes = self.__getBytesFromPlan(right_df, spark)
+
+        # choose 30MB as the cutoff for the broadcast
+        bytes_threshold = 30 * 1024 * 1024
+        if sql_join_opt & (
+            (left_bytes < bytes_threshold) | (right_bytes < bytes_threshold)
+        ):
+            spark.conf.set("spark.databricks.optimizer.rangeJoin.binSize", 60)
+            partition_cols = right_tsdf.partitionCols
+            left_cols = list(set(left_df.columns).difference(set(self.partitionCols)))
+            right_cols = list(
+                set(right_df.columns).difference(set(right_tsdf.partitionCols))
+            )
```

Comment:
Looks like this may be reverting the changes made in #337 @tnixon

Reply:
ugh - not again 🤦🏻‍♂️ 
Thanks for catching this - I'll fix it now...

Reply:
Should be fixed now